src/pages/Upload/components/UploadInfo/UploadInfo.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Description from './Description/Description';
+import PageChangeBtn from './Description/PageChangeBtn';
+import InfoSubmit from './InfoSubmit.js/InfoSubmit';
+import ButtonContainer from './UploadPage/CategoryPage/ButtonContainer';
+import SubmitPage from './UploadPage/CategoryPage/SubmitPage';
+
+const UploadInfo = () => {
+  const [descPage, setDescPage] = useState(false);
+  const [infoPage, setInfoPage] = useState(false);
+
+  const descPageHandler = () => {
+    setDescPage(true);
+  };
+
+  const infoPageHandler = () => {
+    setInfoPage(!infoPage);
+  };
```

Comment:
```suggestion
  const handleDescPageOpen = () => {
    setDescPage(true);
  };

  const toggleInfoPage = () => {
    setInfoPage(!infoPage);
  };
```
함수 이름과 넘겨주고 있는 props의 이름을 이렇게 바꿔주는 건 어떤가요? pageHandler라는 이름만 보고서는 어떤 동작을 하는 함수인지 유추하기 쉽지 않았습니다